### PR TITLE
V2:Astro-915 - Update to Push Button KM Styles

### PR DESCRIPTION
**Pull request by im-cr**
**Ticket**
- https://rocketcom.atlassian.net/browse/ASTRO-915

**Changes** - commit: https://github.com/RocketCommunicationsInc/astro-components/pull/40/commits/da03006f50f48a4465dc23cf1f9708eb7caf3b60
 - rux-push-button.js
 - pushbutton.css
 - theme--dark
 - theme--light

**Design References**
  - [Dark](https://app.abstract.com/projects/2d8cf270-0b93-11e8-89ff-61554354acb1/branches/f22ccbcd-6954-4228-b52f-34e5288e75c8/files/1d17329d-e1a3-490a-a6ad-097e2ecd4825?filter=changed&s=push)
  - [Light](https://app.abstract.com/projects/2d8cf270-0b93-11e8-89ff-61554354acb1/branches/f22ccbcd-6954-4228-b52f-34e5288e75c8/files/b7c2bf0c-9dec-4b07-96d4-ead85393c9d0?filter=changed&s=push)

### CODE REVIEW

File: src/components/rux-push-button/rux-push-button.js [hunk 1]
```diff
@@ -60,29 +60,29 @@ export class RuxPushButton extends LitElement {
...
         .rux-push-button__input:checked + .rux-push-button__button {
           display: flex;
-          color: var(--pushbuttonSelectedTextColor, rgb(91, 255, 0));
-          background-color: var(--pushbuttonSelectedBackgroundColor, rgb(0, 0, 0));
-          border-color: var(--pushbuttonSelectedBorderColor, rgb(0, 0, 0));
+          color: var(--pushbuttonSelectedTextColor);
+          background-color: var(--pushbuttonSelectedBackgroundColor);
+          border-color: var(--pushbuttonSelectedBorderColor);
 
           box-shadow: inset 0 0 4px rgba(0, 0, 0, 0.33);
```
Comment (dmcalester) [inline src/components/rux-push-button/rux-push-button.js:76]: Remove inset shadow

File: src/css/components/pushbutton.css [hunk 2]
```diff
@@ -45,35 +42,32 @@
...
+.rux-pushbutton__input:checked + .rux-pushbutton__button{
   display: flex;
-  color: var(--pushbuttonSelectedTextColor, rgb(91, 255, 0));
-  background-color: var(--pushbuttonSelectedBackgroundColor, rgb(0, 0, 0));
-  border-color: var(--pushbuttonSelectedBorderColor, rgb(0, 0, 0));
+  color: var(--pushbuttonSelectedTextColor);
+  background-color: var(--pushbuttonSelectedBackgroundColor);
+  border-color: var(--pushbuttonSelectedBorderColor);
 
   box-shadow: inset 0 0 4px rgba(0, 0, 0, 0.13);
```
Comment (dmcalester) [inline src/css/components/pushbutton.css:63]: Remove inset shadow

**Review by dmcalester — CHANGES_REQUESTED** [review]
Remove inset shadow from selected state in rux-push-button.js and pushbutton.css

**Comment by im-cr** [comment]
@dmcalester shadows removed

**Comment by amalyah** [comment]
Font size needs to change to 16px equivalent (A5 KM v1.1 body font size)

**Review by amalyah — CHANGES_REQUESTED** [review]
Needs font size change

**Comment by im-cr** [comment]
@amalyah font-size bumped up to 1rem/16px

**Review by amalyah — APPROVED** [review]

**Review by cskerbo — APPROVED** [review]

**Pull request merged**